src/physics_engine.py

```diff
@@ -71,7 +103,7 @@ def run_render_test(self):
         # use pygame for testing
         GRAY = (220, 220, 220)
         pygame.init()
-        size = 640, 240
+        size = 1074, 700
```

Comment:
Could you use `PhysicsEngine.SPACE_WIDTH` and `PhysicsEngine.SPACE_HEIGHT` here?